che-incubator/che-code
# chore: Align with upstream

### What does this PR do?
Rebase against the upstream

### What issues does this PR fix?

https://github.com/eclipse-che/che/issues/23059

### How to test this PR?
Recent changes in the upstream are related to Github authentication, so please test related use cases.
Also: 
- PR jobs should be green
- testing of any use cases is useful as we pick up a lot of commits from the upstream

### Does this PR contain changes that override default upstream Code-OSS behavior?
